client/js/widgets/widget.js

```diff
@@ -43,6 +43,7 @@ export class Widget extends StateManaged {
       borderRadius: null,
       rotation: 0,
       scale: 1,
+      moveLimit: {},
```

Comment:
maybe seeing that this is just limited by drag movements we should change it to dragLimit

Reply:
Good idea.  Done.